v2.0/set-vars.md

```diff
@@ -37,14 +37,16 @@ The variable name is case insensitive. The value can be a list of one or more it
 |---------------|--------------|---------------|----------------------------------------------|
 | `application_name` | The current application name for statistics collection. | Empty string | Yes |
 | `database` | The [current database](sql-name-resolution.html#current-database). | Database in connection string, or empty if not specified | Yes |
-| `default_transaction_isolation` | The default transaction isolation level for the current session. See [Transaction parameters](transactions.html#transaction-parameters) and [`SET TRANSACTION`](set-transaction.html) for more details. | Settings in connection string, or "`SERIALIZABLE`" if not specified  | Yes |
+| `default_transaction_isolation` | <span class="version-tag">New in v2.0:</span> The default transaction isolation level for the current session. See [Transaction parameters](transactions.html#transaction-parameters) and [`SET TRANSACTION`](set-transaction.html) for more details. | Settings in connection string, or "`SERIALIZABLE`" if not specified  | Yes |
+| `default_transaction_read_only` | The default transaction access mode for the current session. See [`SET TRANSACTION`](set-transaction.html) for more details. | `off` | Yes |
```

Comment:
We might want to add a bit more of an explicit description here, e.g., 

> If set to `on`, only read operations are allowed in transactions; if set to `off`, both read and write operations are allowed in transactions.

Thoughts? If we're more explicit here, we should probably be more explicit in the other places, too.